website/tables/gcp/gcp_networkconnectivity_internal_ranges.md

```diff
@@ -1,29 +1,32 @@
 # Table: gcp_networkconnectivity_internal_ranges
 
-This table shows data for GCP Network Connectivity Internal Ranges.
+This table shows data for GCP Networkconnectivity Internal Ranges.
```

Comment:
this can be fixed by adding exception here: https://github.com/cloudquery/cloudquery/blob/main/plugins/source/gcp/resources/plugin/plugin.go#L21

Reply:
done, Thanks